pom.xml

```diff
@@ -88,41 +86,11 @@
     <dependencies>
       <dependency>
         <groupId>io.jenkins.tools.bom</groupId>
-        <artifactId>bom-2.289.x</artifactId>
-        <version>1135.va_4eeca_ea_21c1</version>
+        <artifactId>bom-2.361.x</artifactId>
+        <version>1798.vc671fe94856f</version>
         <scope>import</scope>
         <type>pom</type>
       </dependency>
-      <dependency>
-        <groupId>org.jenkins-ci.plugins.to-declarative</groupId>
-        <artifactId>declarative-pipeline-migration-assistant-api</artifactId>
-        <version>${project.version}</version>
-      </dependency>
-      <dependency>
-        <groupId>org.apache.commons</groupId>
-        <artifactId>commons-lang3</artifactId>
-        <version>3.12.0</version>
-      </dependency>
-      <dependency>
-        <groupId>org.jenkins-ci.plugins.workflow</groupId>
-        <artifactId>workflow-step-api</artifactId>
-        <version>622.vb_8e7c15b_c95a_</version>
-      </dependency>
-      <dependency>
-        <groupId>com.atlassian.plugins</groupId>
-        <artifactId>atlassian-plugins-core</artifactId>
-        <version>4.0.0-m029</version>
-      </dependency>
-      <dependency>
-        <groupId>org.codehaus.plexus</groupId>
-        <artifactId>plexus-utils</artifactId>
-        <version>3.3.0</version>
-      </dependency>
-      <dependency>
-        <groupId>io.atlassian.fugue</groupId>
-        <artifactId>fugue</artifactId>
-        <version>4.7.2</version>
-      </dependency>
```

Comment:
Unneeded and probably just there to satisfy Enforcer from a time before the BOM existed.